File: C#/oddities.cs
using System;

public class oddities {
    public static void Main (string[] args) {
        int cases = int.Parse (Console.ReadLine ());
        while (cases-- > 0) {
            int num = int.Parse (Console.ReadLine ());
            Console.WriteLine ((Math.Abs (num) & 1) == 0 ? num + " is even" : num + " is odd");
        }
    }
}

File: Java/oddities.java

//  https://open.kattis.com/problems/oddities
import java.util.Scanner;

public class oddities {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int cases = sc.nextInt();
        while (cases-- > 0) {
            int num = sc.nextInt();
            System.out.println(num % 2 == 0 ? num + " is even" : num + " is odd");
        }
        sc.close();
    }
}